### PR TITLE
Fix website not using correct localized icon title.

### PR DESCRIPTION
**Pull request by kj415j45**
Live version:
image

After fix:
image

### CODE REVIEW

**Review by adamrusted — APPROVED** [review]

**Comment by adamrusted** [comment]
Thanks for the work on this @kj415j45 - will wait for @mondeja to sign-off on website changed, but looks good to me. 😄

**Review by mondeja — APPROVED** [review]

**Pull request merged**
